core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1047,9 +1031,8 @@ class KafkaApis(val requestChannel: RequestChannel,
   }
 
   def close() {
-    quotaManagers.foreach { case (apiKey, quotaManager) =>
-      quotaManager.shutdown()
-    }
+    quotas.produce.shutdown()
+    quotas.fetch.shutdown()
```

Comment:
Wouldn't it be better to have a `shutdown()` in `quotas` that closes all of them?